chsh_inequality.py

```diff
@@ -0,0 +1,21 @@
+# Samle code from: https://learning.quantum.ibm.com/tutorial/chsh-inequality
+
+import numpy as np
+
+# Qiskit imports
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.quantum_info import SparsePauliOp
+
+# Qiskit Runtime imports
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+# Plotting routines
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tck
+
+# To run on hardware, select the backend with the fewest number of jobs in the queue
+service = QiskitRuntimeService(channel="ibm_quantum")
+backend = service.least_busy(operational=True, simulator=False, min_num_qubits=127)
```

Comment:
Could you set the minimum number of qubits to a more useful value for the Bell’s inequality problem?